internal/events/eventer.go

```diff
@@ -48,6 +50,7 @@ const (
 
 	GoChannelDriver = "go-channel"
 	SQLDriver       = "sql"
+	BuiltKey        = "built_at"
```

Comment:
Can you add some docs on what `BuiltKey` is supposed to be? Is that the time that the message was built?

Reply:
Ah! I see now, it's the time when the message was published.

Reply:
@eleftherias should this be `published_at`?